frontend/src/components/layout/Layout.tsx

```diff
@@ -138,6 +140,13 @@ export const Layout = (props: Props) => {
           runtimeData={props.runtimeData}
         />
         <header className={styles["header-outer"]}>
+          {isFlagActive(props.runtimeData, "holiday_promo_2023") &&
```

Comment:
Maybe also add "is today no later than Dec 31, 2023" as a condition? So it auto-hides when it's no longer available.